File: Witcher-front/src/Components/Sword.js
import React from 'react';

const divStyle = {
    width: '300px',
    border: '2px #000',
    borderRadius: '10px'
};

class Sword extends React.Component {


    render() {
        return(
            <div style={divStyle}>
                {this.props.array.map((elem) => {
                    //if (elem.armorName === this.props.name)
                        return (
                            <div>
                                <table>
                                    <tr>
                                <td>Название - {elem.swordName}</td>
                                <td>Атака - {elem.atack}</td>
                                <td>Цена - {elem.price}</td>
                                    </tr>
                                </table>
                            </div>);
                })}
            </div>
        );
    }

}

export default Sword;

File: Witcher-back/src/main/java/web/witcher/Entities/Sword.java
package web.witcher.Entities;



import lombok.Data;

import javax.persistence.*;

@Entity
@Data
@Table(name = "Sword")
public class Sword {

    @Id
    @Column(name = "swordName")
    private String swordName;

    @Column(name = "price")
    private int price;

    @Column(name = "atack")
    private int atack;

}
